pkg/detectors/maxmindlicense/v2/maxmindlicense_v2.go

```diff
@@ -37,49 +36,44 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	dataStr := string(data)
 
 	keyMatches := keyPat.FindAllStringSubmatch(dataStr, -1)
-	idMatches := idPat.FindAllStringSubmatch(dataStr, -1)
 
 	for _, keyMatch := range keyMatches {
 		keyRes := strings.TrimSpace(keyMatch[1])
 
-		for _, idMatch := range idMatches {
-			if len(idMatch) != 2 {
+		r := detectors.Result{
+			DetectorType: detectorspb.DetectorType_MaxMindLicense,
+			Redacted:     keyRes,
+			Raw:          []byte(keyRes),
+			RawV2:        []byte(keyRes),
```

Comment:
I don't think `RawV2` needs to be set if it will always be the same as `Raw`

https://github.com/trufflesecurity/trufflehog/blob/e66bdfb7f911eafd465cad1d6a1bdc3462663c59/pkg/detectors/detectors.go#L53-L55

It appears other detectors that do not use an "id" also do not generally set `RawV2`